packages/ai-jsx/src/batteries/use-tools.tsx

```diff
@@ -122,8 +122,6 @@ export interface UseToolsProps {
  *
  * This is conceptually similar to [chatGPT plugins](https://openai.com/blog/chatgpt-plugins).
  *
- * @see ../../../examples/src/bakeoff/zepp-health/zepp.tsx
```

Comment:
We should still link to an example of this being used.

Reply:
Okay, added everything back, but renamed it to a generic name: `health-agent`.